playbooks/azure/openshift-cluster/tasks/provision_instance.yml

```diff
@@ -25,7 +25,7 @@
     az vm create
     -n vm
     -g "{{ openshift_azure_resource_group_name }}"
-    --size Standard_D4s_v3
+    --size "{{ openshift_azure_vm_size | default('Standard_D4s_v3') }}"
```

Comment:
think I'd doc openshift_azure_vm_size otherwise lgtm

Reply:
updated the doc, thanks.